VimeoNetworking/Sources/Request+Notifications.swift

```diff
@@ -8,14 +8,18 @@
 
 public extension Request
 {
-    private static var Path: String { return "/me/notifications/subscriptions" }
+    private typealias ParameterDictionary = [String: AnyObject]
+    
+    private static var Path: String { return "/me/notifications" }
```

Comment:
This URI should be pulled off the user model `user > connections > notifications > uri` rather than being hard-coded